sdk/sdktypes/connection.go

```diff
@@ -28,11 +28,7 @@ func (ConnectionTraits) Validate(m *ConnectionPB) error {
 }
 
 func (ConnectionTraits) StrictValidate(m *ConnectionPB) error {
-	var errs []error = []error{mandatory("name", m.Name)}
-	if m.ConnectionId != BuiltinSchedulerConnectionID.String() {
-		errs = append(errs, mandatory("project_id", m.ProjectId), mandatory("integration_id", m.IntegrationId))
-	}
-	return errors.Join(errs...)
+	return errors.Join(mandatory("name", m.Name))
```

Comment:
for non-scheduler connection project_id and integration_id were mandatory. Why did u remove them?

Reply:
connection_id cannot be mandatory as a connection is given to the service with no id.
integration_id need to be mandatory though - will re-add it.

Reply:
ah project_id, right. yes - readding both.